packages/altair-koa-middleware/package.json

```diff
@@ -15,15 +15,15 @@
     "@types/supertest": "^2.0.11",
     "jest": "29.4.1",
     "koa": "^2.13.1",
+    "koa-send": "^5.0.1",
```

Comment:
Why was added to dev dependencies, instead of dependencies? It's used in the published code.